src/main/java/com/skynest/uitesting/pages/RegistrationForm.java

```diff
@@ -0,0 +1,64 @@
+package com.skynest.uitesting.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class RegistrationForm {
+    private final WebDriver driver;
+    private final WebDriverWait wait;
+
+    private final By firstNameSelector = By.id("firstNameInput");
+    private final By lastNameSelector = By.id("lastNameInput");
+    private final By emailAddressSelector = By.id("emailInput");
+    private final By phoneNumberSelector = By.id("phoneInput");
+    private final By homeAddressSelector = By.id("adressInput");
```

Comment:
There seems to be a typo here ("adressInput"); either in this automation suite or on the front end